app/views/mortgage_calculator/repayments/_tabs.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="tabs">
+      <dl class="tabs__list rendered-from-js">
+        <dt><a href="#" class="tabs__link" ng-click="showMonthly($event)" ng-class="{ 'tabs__current' : viewMonthlyRepayments }"><%= I18n.t("mortgage_calculator.repayment.tabs.repayment.title") %></a></dt>
+        <dt><a href="#" class="tabs__link" ng-click="showInterest($event)" ng-class="{ 'tabs__current' : viewInterestRepayments }"><%= I18n.t("mortgage_calculator.repayment.tabs.interest_only.title") %></a></dt>
+      </dl>
+      <div class="tabs__content">
+        <div class="tabs__panel" ng-show="viewMonthlyRepayments" id="panel__monthly_repayments">
```

Comment:
`id` attributes being used here. Is there any way we could/should be using classes instead?